packages/mason_logger/lib/src/mason_logger.dart

```diff
@@ -53,7 +53,17 @@ class Logger {
   void delayed(String? message) => _queue.add(message);
 
   /// Writes progress message to stdout.
-  Progress progress(String message) => Progress._(message, _stdout, level);
+  /// Provide a animation list via [progressAnimation].
+  Progress progress(
+    String message, {
+    List<String>? progressAnimation,
```

Comment:
Should there also be a way to configure the progress animation for the logger instance? I feel like it would be quite cumbersome to have to constantly provide a custom `progressAnimation` to each `progress` call. wdyt?

Reply:
Yup, I agree with this. I will have a look at this soon!

Reply:
Added `progressAnimation` as an optional parameter to the logger instance. The developer can also override each progress call with another animation if they wanted to.

Let me know if there are any other issues!